### DISCUSSION TITLE
Start Reaper transport and Playtime transport together

### DISCUSSION THREAD

**Discussion by andrasf** [helgoboss/helgobox#1327] [General]
Hi,

I'm not sure if this a bug or not. I use stuff like sequencers that need Reaper transport to run. In this case I can start Playtime and Reaper transport together by pressing space in Reaper window.

But then if I open a midi clip to edit in Playtime, playback often stops. Either at the end of the clip, or sometimes earlier.

How are the supposed to work together? Can I start and keep running both?

Thanks,
Andras

**Comment by helgoboss** [comment]
There's a setting "Transport sync" in the matrix settings, which you can turn to "Full" in order to automatically start REAPER playback when you press SPACE while Playtime is focused or press the toolbar play button in Playtime.

Can you explain in more detail what you mean by "playback often stops" (is it just the clip that stops playing or is it REAPER that stops, etc.)? Or send a video which shows this? I think I never experienced this.

**Comment by andrasf** [comment]
Yes, like this: https://www.youtube.com/watch?v=atSpjXRXzro

It seems like it works fine on the second start.

**Comment by andrasf** [comment]
Or, it's fine if I start the scene and not the transport. I was annoyed first, because I started playback by pressing space. And basically it seems like this happens if Reaper playback is started, either with full sync or by pressing space in the Reaper window in partial sync mode.

**Comment by helgoboss** [comment]
Interesting. So basically REAPER simply stops playback, seemingly out of the blue. At the top of my head, only one thing comes to my mind:

REAPER preferences => Audio => Playback has the following checkboxes:
<img width="284" alt="image" src="https://github.com/user-attachments/assets/1bc0f927-72c4-4f08-b7ae-3e646de51d32">

1. Is one of them enabled?
2. If yes, does the issue still occur if you disable both of them?

If this is the issue (and also if not hehe) I think I should transparently disable those options while Playtime is running. I remember I did that for Playtime 1, in which it was even more important.

**Comment by andrasf** [comment]
Bingo, first one was enabled, it works after disabling. Very cool, thanks!
